testsuite/gnat2goto/tests/type_modular/modular.adb

```diff
@@ -6,7 +6,12 @@ procedure Modular is
   type Two_Bit is mod 4;
   function Next (X : Two_Bit) return Two_Bit is (X + 1);
   Number : constant Two_Bit := 3;
+
+  type Unsigned_64 is mod 2**64;
+  Zero_Unsigned : constant Unsigned_64 := 0;
```

Comment:
Would you mind renaming these and using a `Next` function like for the other cases for consistencies sake?

Reply:
None of these add any value what so ever. For a `Next` function, I'm not going to use it in any way, and to make it prose `Unsigned_Sixty_Four_Bit_Value` seems annoying.